### DISCUSSION TITLE
Micropython support?

### DISCUSSION THREAD

**Discussion by andywestley** [iamh2o/rgbw_colorspace_converter#14] [Q&A]
Hi there,

I'm new to Python and Micropython, and have stumbled across this project while trying to solve the problem of RGB to RGBW conversionwhile trying to build a web interface for a Raspberry Pi Pico-powered RGBW LED installation.

As I said I'm not so familiar with what is missing from standard python to make it micro-sized, but do you know whether there is a specific limitation that would prevent this or a version of this being deployed in a Micropython environment? I did just try to deploy to my Pico device, and it wouldn't even install.

**Comment by iamh2o** [comment]
I'm not familiar enough with micropython to know where things might be falling down.  Can you post the error message when trying to install?  My guess would be a dependency, probably `colr`.  I might suggest going through the requirements of this repo and trying to install each on their own to see if any of those fail.  This repo can be used w/out color... if that is the problem, detangling that from the build may be all that is needed to do colorspace conversions.
